src/rprblender/ui/render.py

```diff
@@ -170,6 +170,8 @@ def draw(self, context):
 
         col.prop(settings, 'use_gl_interop')
 
+        col.prop(settings, 'viewport_resolution_upscale')
```

Comment:
Maybe it should be hidden or disabled for Legacy and Hybrid modes?

Reply:
It is working in all modes.